backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/po/CustomColumnService.java

```diff
@@ -22,87 +22,55 @@
 
 package de.metas.po;
 
-import com.google.common.collect.ImmutableMap;
-import de.metas.i18n.AdMessageKey;
 import de.metas.organization.IOrgDAO;
 import de.metas.organization.OrgId;
 import de.metas.util.Services;
-import de.metas.util.converter.POValueConverters;
 import lombok.NonNull;
-import org.adempiere.exceptions.AdempiereException;
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.adempiere.util.lang.impl.TableRecordReference;
 import org.compiere.model.PO;
-import org.compiere.model.POInfo;
-import org.compiere.model.POInfoColumn;
 import org.springframework.stereotype.Service;
 
-import javax.annotation.Nullable;
 import java.time.ZoneId;
 import java.util.Map;
-import java.util.Optional;
 
 @Service
 public class CustomColumnService
```

Comment:
i think this service shall be in `org.adempiere.ad.persistence` package, near TableModelLoader.